src/AppFrame.tsx

```diff
@@ -7,22 +7,87 @@ import { Routes } from "./Routes";
 import { TwitterTimelineEmbed } from "react-twitter-embed";
 import { useTranslation } from "react-i18next";
 import i18next from "i18next";
+import { ItemProps } from "@shopify/polaris/types/latest/src/components/Navigation/components";
+
+const languages = [
+  {
+    langCode: "en-CA",
+    label: "English",
+  },
+  {
+    langCode: "fr-CA",
+    label: "Français",
+  },
+  {
+    langCode: "ar",
+    label: "العربية",
+  },
+  /*
+  {
+    langCode: "de",
+    label: "Deutsch",
+  },
+  {
+    langCode: "es",
+    label: "Español",
+  },
+  {
+    langCode: "pa-PK",
+    label: "پن٘جابی",
+  },
+  {
+    langCode: "pa-IN",
+    label: "ਪੰਜਾਬੀ",
+  },
+  */
+  {
+    langCode: "pl-PL",
+    label: "Polski",
+  },
+  {
+    langCode: "ru-RU",
+    label: "Русский",
+  },
+  /*
+  {
+    langCode: "pt",
+    label: "Português",
+  },
+  */
+  {
+    langCode: "zh-CN",
+    /* label: "简体中文", */
+    label: "中文",
+  },
+  /*
+  {
+    langCode: "zh-TW",
+    label: "繁體中文",
+  },
+  */
+];
+
+function LanguageBar(): ItemProps[] {
+  const output: ItemProps[] = languages.map((x) => {
```

Comment:
Minor, but my preference is for variables to have descriptive names, so instead of `x` I would use `language`

Reply:
changed